### ISSUE TITLE
Source model and prediction results

### ISSUE THREAD

**Issue by kbuika** [bhavesh112/PlantDiseasePrediction-API#1]
Hello @bhavesh112,

Really good project set-up. Even without documentation, I was able to set it up. However, I have a question about the prediction results and the source model. Could you please send me some resources on the source model and what the expected predictions are?

I will highly appreciate that. Thank you.

**Comment by bhavesh112** [comment]
Hello @kibuikaCodes 
You can refer this [dataset](https://www.kaggle.com/datasets/vipoooool/new-plant-diseases-dataset). For information about the model you can refer this [notebook](https://colab.research.google.com/drive/1Truh15sNFEA0oCgiv--O0pNl0l5fQ-fR). I have used VGG-19 model from keras. The accuracy is not that great but it works with good images. For the different classes that can be predicted you can check this [file](https://github.com/bhavesh112/PlantDiseasePrediction-Frontend/blob/master/src/utils.ts).

**Comment by kbuika** [comment]
Thank you very much.
